### PR TITLE
[19.07]syncthing: update to 1.13.1

### PR DESCRIPTION
**Pull request by aparcar**
https://github.com/syncthing/syncthing/releases/tag/v1.14.0

Signed-off-by: Paul Spooren <mail@aparcar.org>
(cherry picked from commit 2ce92152133c7c8b956c9738cb1ed7d05ca6b10e)

### CODE REVIEW

File: utils/syncthing/Makefile [hunk 1]
```diff
@@ -1,12 +1,11 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=syncthing
```
Comment (neheb) [inline utils/syncthing/Makefile:3]: Why is this gone?

Comment (aparcar) [reply utils/syncthing/Makefile:3]: I wanted to test builds via the backported CI but didn't get around to do so yet

**Comment by Neustradamus** [comment]
@aparcar: 1.14.0 or other?
- https://github.com/openwrt/packages/pull/15102#issue-591269005

**Comment by neheb** [comment]
```
 note: module requires Go 1.14
```

**Comment by BKPepe** [comment]
Closing. It is kinda old

**Pull request closed without merging**